src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/Macros/JoinMacroConfig.cs

```diff
@@ -35,12 +35,12 @@ internal JoinMacroConfig(JoinMacro macro, IGeneratedSymbolConfig generatedSymbol
 
             foreach (JToken entry in jArray)
             {
-                if (entry is not JObject jobj)
+                if (entry is not JObject jObject)
```

Comment:
Not sure about this. `jobj` seems fine. `jObject` and `JObject` are too similar I think

Reply:
I converted it to jObj. (o being small where spell check started failing)